package/lib-edgetpu/.cm/meta.json

```diff
@@ -0,0 +1,49 @@
+{
+  "customize": {
+    "extra_dir": "",
+    "install_env": {
+      "PACKAGE_URL": "https://packages.cloud.google.com/apt/pool/",
+      "PACKAGE_WGET": "YES",
+      "PACKAGE_WGET_EXTRA": "--no-cookies"
+    },
+    "version": "with-variations"
+  },
+  "end_full_path": {
+    "linux": "usr/lib/aarch64-linux-gnu/libedgetpu.so.1"
+  },
+  "need_cpu_info": "yes",
+  "only_for_host_os_tags": [
+    "linux"
+  ],
+  "only_for_target_os_tags": [
+    "linux"
+  ],
+  "process_script": "install",
+  "soft_uoa": "2783dce06e383615",
+  "suggested_path": "lib-edgetpu",
+  "tags": [
+    "lib",
+    "libedgetpu",
+    "edgetpu"
+  ],
+  "variations": {
+    "v14.1_std": {
```

Comment:
I suggest to call it `std_14.1_arm64` as in the file name. I recommend also to create a complete copy under a shorter name `std`, which we will use by default and update as new versions come along.

Reply:
Ok. Done.